gulpfile.js

```diff
@@ -27,7 +31,7 @@ require('gulp-task-loader')({
 // Define task for each theme, locale, lang, processing type etc.
 // Gulp can't run same task in parallel, so we need different names
 Object.keys(config.themes).forEach(name => {
-  var theme = config.themes[name];
+  const theme = config.themes[name];
   theme.locale.forEach(locale => {
     gulp.task(
       theme.lang + ':' + name + ':' + locale,
```

Comment:
@Igloczek are these tasks used anywhere? Removing config.themes from the gulpfile.js will cause this to stop functioning